### PR TITLE
Add batch map APIs in Doxygen documentation

### PR DESCRIPTION
**Pull request by shpalani**
## Description

This PR contains the doxygen support for the following APIs:
- bpf_map_lookup_batch
- bpf_map_lookup_and_delete_batch
- bpf_map_delete_batch
- bpf_map_update_batch

## Testing

_Do any existing tests cover this change? Yes
Are new tests needed? No

## Documentation

CICD generates doxygen documentation in docs/html

## Installation

_Is there any installer impact for this change? No

### CODE REVIEW

File: include/bpf/bpf.h [hunk 1]
```diff
@@ -186,6 +186,112 @@ bpf_map_lookup_elem(int fd, const void* key, void* value);
 int
 bpf_map_update_elem(int fd, const void* key, const void* value, __u64 flags);
 
+/**
+ * @brief Look up multiple entries in a BPF map in batch.
```
Comment (dthaler) [inline include/bpf/bpf.h:190]: I'm noticing the actual English text seems to be gratuitously different from the libbpf doxygen docs at https://github.com/libbpf/libbpf/blob/master/src/bpf.h#L191

Why the difference?

Comment (shpalani) [reply include/bpf/bpf.h:190]: The contents in ebpf-for-windows/include/bpf/bpf.h comments are different from the libbpf doc as pointed out. I don't know the reason for deviating. Should the _batch APIs comment follow the libbpf format?

Comment (shpalani) [reply include/bpf/bpf.h:190]: As discussed in today's ebpf-for-windows public office hours triage,
 - Keep the same content as in libbpf.
  - Remove exception keywords.
  - File a separate task to clean up the Doxygen file for all the APIs to match libbpf format.

Comment (shpalani) [reply include/bpf/bpf.h]: Fixed.

**Review by dthaler — APPROVED** [review]
-

**Review by saxena-anurag — APPROVED** [review]

**Pull request merged**